plugins/chat/assets/javascripts/discourse/widgets/chat-mention-notification-item.js

```diff
@@ -44,10 +44,13 @@ const chatNotificationItem = {
   },
 
   url(data) {
-    const title = data.chat_channel_title
-      ? slugifyChannel(data.chat_channel_title)
-      : "-";
-    return `/chat/channel/${data.chat_channel_id}/${title}?messageId=${data.chat_message_id}`;
+    const slug = slugifyChannel({
```

Comment:
I we can't fix it by a default server side as talked above, we could also change client side `slugifyChannel` to always return `-` if nothing else.

Reply:
Also makes sense :+1: